### PR TITLE
Ports Bay bone break sounds

### PR DESCRIPTION
**Pull request by jknpj**
Now you can REALLY hear that sickening crack:

http://vocaroo.com/i/s0YYyfAIkxgp
http://vocaroo.com/i/s09uggXSCS1S
http://vocaroo.com/i/s0Kp32NoF4CO
http://vocaroo.com/i/s0l0ETGgZyT8

:cl: 
 * soundadd: Adds four sounds that are randomly played when bones break.

### CODE REVIEW

**Comment by ultraplen** [comment]
Please add a vocaroo preview

**Comment by ghost** [comment]
Now you'll know when to stop harmbatonning the clowns hands!

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -777,6 +777,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(owner.feels_pain())
 		owner.emote("scream", , , 1)
 
+	playsound(owner.loc, "fracture", 100, 1, -2)
```
Comment (Probe1) [inline code/modules/organs/organ_external.dm:780]: loud

**Comment by Probe1** [comment]
Like I'm gonna stop when his hands break.

**Comment by Shadowmech88** [comment]
What he meant to say is, now you'll know when to start beating a different limb.

**Comment by yclatious** [comment]
But don't people already scream when they get their bones broken? That's already a tell when its time to beat another limb.

**Comment by SkowronX** [comment]
It adds to the effect, which is nice. More sound effects like this are a plus.

**Comment by jknpj** [comment]
@yclatious if they somehow don't feel pain they won't scream

File: code/modules/organs/organ_external.dm (hunk 1 above)
Comment (jknpj) [reply code/modules/organs/organ_external.dm:780]: Actually not that loud:
https://webmshare.com/ReXKA

Comment (Probe1) [reply code/modules/organs/organ_external.dm:780]: Really sounds more like a click than a snap.

**Pull request merged**